frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -53,6 +57,9 @@ pods:
           ssl-server:
             template: ssl-server.xml
             dest: {{HDFS_VERSION}}/etc/hadoop/ssl-server.xml
+          ssl-client:
+            template: {{CONFIG_TEMPLATE_PATH}}/ssl-client.xml
```

Comment:
@nickbp made some changes w.r.t `{{CONFIG_TEMPLATE_PATH}}` in #1539. It seems as if this is not needed anymore, but I would check with him to be sure.

Reply:
(Looking at the `ssl-server` section just above this seems to confirm that this is no longer needed).

Reply:
Yes, relative paths can be provided now, and they will default to the directory that the svc.yml is in (can be customized in the java API)

Reply:
@nickbp  the template is still used jn Cassandra. Should this be the case? If not, I can create a PR tomorrow to remove it too.

Reply:
Oh interesting, would've expected that to not work. It's not being set by cassandra's marathon.json.mustache anymore... (yes it can be removed from cassandra)